src/server/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { parse } from 'pg-connection-string'
+import { FastifyRequest } from 'fastify'
+
+export const extractRequestForLogging = (request: FastifyRequest) => ({
+  method: request.method,
+  url: request.url,
+  pg: parse(request.headers.pg as string).host,
+  body: request.body,
```

Comment:
Hmm I'm not sure if it's OK to log the body by default. Aside from concerns about the size of the object that might get logged, we might also end up logging sensitive data here.

If needed, we could log a hash of the body?

Reply:
:+1: removed `body` for now